tools/cluster_mgr.py

```diff
@@ -330,30 +369,60 @@ def shutdown(args):
 def main():
     parser = argparse.ArgumentParser(
         description="""
-Local Cluster Manager
+Dragonfly Manual Cluster Manager
+
+This tool helps managing a Dragonfly cluster manually.
+Cluster can either be local or remote:
+- Starting Dragonfly instances must be done locally, binary is assumed to be under ../build-opt
+- Remote Dragonflies must already be started, and initialized with `--cluster_mode=yes`
 
 Example usage:
 
-Create a 3+3 nodes cluster:
-  ./cluster_mgr.py --action=create --num_masters=3 --replicas_per_master=1
+Create a 3 node cluster locally:
+  ./cluster_mgr.py --action=create_locally --num_masters=3
+This will create 3 Dragonfly processes with ports 7001-7003.
+Ports can be overridden with `--first_port`.
 
-Connect to cluster and print current config:
-  ./cluster_mgr.py --action=print
+Create a 6 node cluster locally, 3 of them masters with 1 replica each:
+  ./cluster_mgr.py --action=create_locally --num_masters=3 --replicas_per_master=1
 
-Connect to cluster and shutdown all nodes:
-  ./cluster_mgr.py --action=shutdown
+Connect to existing cluster and print current config:
+  ./cluster_mgr.py --action=print_config
+This will connect to localhost:6379 by default. Override with `--target_host` and `--target_port`
 
-Connect to cluster and move slots 10-20 to master with port 7002:
-  ./cluster_mgr.py --action=move --slot_start=10 --slot_end=20 --new_owner=7002
+Configure an existing Dragonfly server to be a standalone cluster (owning all slots):
+  ./cluster_mgr.py --action=config_single_remote
+This connects to an *existing* Dragonfly server, and pushes a config telling it to own all slots.
+This will connect to localhost:6379 by default. Override with `--target_host` and `--target_port`
 
-Migrate slots 10-20 to master with port 7002
-  ./cluster_mgr.py --action=migrate --slot_start=10 --slot_end=20 --new_owner=7002
-"""
+Attach an existing Dragonfly server to an existing cluster (owning no slots):
+  ./cluster_mgr.py --action=attach --attach_host=HOST --attach_port=PORT
+This will connect to existing cluster present at localhost:6379 by default. Override with
+`--target_host` and `--target_port`
+
+To set up a new cluster - start the servers and then use
+  ./cluster_mgr.py --action=config_single_remote ...
```

Comment:
I want to discuss the api here to build the remote cluster but this is working now and its ok for the demo so LGTM

Reply:
Sure.. let's discuss when you're free :)
But can I haz approval?